charts/dapr/charts/dapr_config/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Dapr configuration
+name: dapr_config
+version: 0.1.0
```

Comment:
Shouldn't we bump up the chart version?

Reply:
Yes. Can you commit directly? I'm a little tied up..

Reply:
Let's do it as a separate pr.